pilot/pkg/serviceregistry/kube/controller/multicluster.go

```diff
@@ -48,6 +46,7 @@ type Multicluster struct {
 	m                     sync.Mutex // protects remoteKubeControllers
 	remoteKubeControllers map[string]*kubeController
 	networksWatcher       mesh.NetworksWatcher
+	fetchcaRoot           func() map[string]string
```

Comment:
Can you add a comment for what the key represents in the map[string]string structure?

Reply:
added.

Reply:
Where is the added comment?

Reply:
will push soon :) . ```+       // fetchCaRoot maps the certName to cert
+       fetchCaRoot           func() map[string]string```